DogTales/dog/player/player.cpp

```diff
@@ -1,19 +1,24 @@
-#include <dog/player.hpp>
+#include <dog/player/player.hpp>
 
 namespace dog {
 Player::Player(bave::App& app, glm::vec2 const world_space) : m_app(app), m_world_space(world_space) {
 	m_sprite.set_size(size_v);
+
+	m_player_controller.bind_throttle("move_x", {.lo = bave::Key::eLeft, .hi = bave::Key::eRight});
+	m_player_controller.bind_throttle("move_x", {.lo = bave::Key::eA, .hi = bave::Key::eD});
+	m_player_controller.bind_throttle("move_y", {.lo = bave::Key::eDown, .hi = bave::Key::eUp});
+	m_player_controller.bind_throttle("move_y", {.lo = bave::Key::eS, .hi = bave::Key::eW});
+	m_player_controller.bind_key("jump", bave::Key::eSpace);
```

Comment:
If we wanted to change these in-game (say, settings menu > key binds), how would we go about that? would we have to "unbind" the throttles and keys?

Reply:
Correct, we would need a way to remove existing bindings as this class is developed.